test/helpers/setup-codeshift.js

```diff
@@ -0,0 +1,5 @@
+import { codeshift } from '../../src/utils';
+
+(async function() {
+  await codeshift.js?.install();
+})();
```

Comment:
Since we're not awaiting on the iife, does mocha wait on global pending promises before running tests? Should this be inside of a `before` hook (still in this separate file) to ensure the tests don't start before the install is done? I believe mocha documents global hooks like that anyhow (a separate file with the hooks that is required before all tests).